src/app.config.js

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 const Config = {
+  // TEMP: Base service urls
+  baseUrls: {
+    iotHubManager: "https://YOURHOSTNAME/iothubmanager/v1/"
```

Comment:
If you are going to keep the baseUrls here I could make the hostname a variable.

```
const baseURL = '';
...
const Config = {
    serviceEndpoints: {
        iotHubManager: `${baseURL}iothubmanager/v1/`
    }
    ...
}
```

Reply:
done.